libs/application/templates/parental-leave/README.md

```diff
@@ -1,5 +1,138 @@
+
+#### Employer waiting to assign
+
+If the applicant is employed by an employer (not self employed), then they are asked to provide an email address which a confirmation email will be sent to. When the employer receives the email they can click a link which automatically assigns them to the application and advances it to the next state
+
+#### Employer approval
+
+Here the employer will have a chance to review the periods selected by the applicant as well as pension fund and union.
+
+The employer can then approve or request changes
+
+#### Vinnumalastofnun approval
+
+Applications that have been sent to VMST
+
+### Parental leave template API module
+
+For async actions that require server side logic we have the template api modules. The module for this application contains email templates as well as external actions like `sendApplication`.
+
+Have a look at `libs/application/template-api-modules/src/lib/modules/templates/parental-leave/parental-leave.module.ts` for further information on these actions, and `libs/application/template-api-modules/README.md` to get up to speed on template api modules.
+
+### API and X-Road
+
+All user interaction goes through our GraphQL API (`api`) which then handles communicating with other APIs.
+
+To communicate with VMST a request has to go through [X-Road](https://docs.devland.is/technical-overview/x-road). Both in development and on production environments.
+
+To connect to VMST test API you'll want to start the local proxy, following these steps:
+
+- Open AWS: https://island-is.awsapps.com/start#/
+  - island-is-development01 > AWSPowerUserAccess > Command line access
+- Choose an option you prefer in the modal
+- Connect to the proxy `./scripts/run-xroad-proxy.sh`
+
+See `libs/api/domains/directorate-of-labour/src/lib/directorate-of-labour.module.ts` for examples of VMST communication.
+
+### Contacts
+
+VMST API specific questions:
+[Guðmundur K. G. Kolka](mailto:gudmundur.kolka@advania.is), API and NAV developer
+
+VMST Devops:
+[Unnar Sigurðsson](mailto:unnar.sigurdsson@vmst.is), VMST API dev ops and x-road integration
+
+We also have two channels on the islandis slack #faedingarorlof-profanir and #faedingarorlof
+
+### Localisation
+
+All localisation can be found on Contentful.
+
+- [Parental leave application translations](https://app.contentful.com/spaces/8k0h54kbe6bj/entries/pl.application)
+- [Application system translations](https://app.contentful.com/spaces/8k0h54kbe6bj/entries/application.system)
+
+⚠️ When creating new text strings in the messages.ts file for the application, be sure to update Contentful, this is done using the command:
+
+```
+CONTENTFUL_MANAGEMENT_ACCESS_TOKEN={secret*} yarn nx run application-templates-parental-leave:extract-strings
+```
+
+_secret is found within AWS parameter store_
+
+See [link](https://docs.devland.is/apps/web/subpages#contentful) for further documentation for localisation in island.is repo.
```

Comment:
Bad link?

Maybe link to [this](https://docs.devland.is/libs/localization#message-extraction) and leave out the CONTENTFUL_MANAGEMENT_ACCESS_TOKEN stuff.